src/Components/Buttons.mdx

```diff
@@ -19,92 +19,119 @@ import { Playground } from 'docz';
+### Outline (Secondary) Button 
+
+- Use when the pages primary action is not obvious or when there are complimentary actions the user can perform.
+- There can be multiple outline/secondary buttons on a single page.
 
 <Playground>
   <ButtonToolbar>
-    <Button>Secondary</Button>
+    <Button>Outline</Button>
+    <Button active>Active</Button>
+    <Button disabled>Disabled</Button>
   </ButtonToolbar>
 </Playground>
 
-**Destruction** - Destructive buttons express caution to users and are tied to actions like delete or deactivate
+### Link Button
+
+- Use when trying to give an action the least possible emphasis.
+- Use when there are many actions available and you are trying not to saturate the page with buttons. 
+- Link buttons are not hyperlinks in a pararaph of text, they are to be used like buttons.
+- There can be multiple link buttons on a single page.
 
 <Playground>
   <ButtonToolbar>
-    <Button bsStyle="danger">Destructive</Button>
+    <Button bsStyle="link">Standard</Button>
+    <Button bsStyle="link" active>Active</Button>
+    <Button bsStyle="link" disabled>Disabled</Button>
   </ButtonToolbar>
 </Playground>
 
-**Text** - Text buttons emphasize actions that are the least important
+### Destructive Button
+
+- Use whenever a destructive action is present on a screen. 
+- These buttons are typically used in modals whereby the user must confirm the destructive action.
 
 <Playground>
   <ButtonToolbar>
-    <Button bsStyle="link">Link</Button>
+    <Button bsStyle="danger">Danger</Button>
+    <Button bsStyle="danger" active>Active</Button>
+    <Button bsStyle="danger" disabled>Disabled</Button>
   </ButtonToolbar>
 </Playground>
 
-**Small** - Small buttons \_\_\_
+### Button Sizing
+
+<Playground>
+  <ButtonToolbar>
+    <Button bsStyle="primary" bsSize="small">Small Button</Button>
+    <Button bsSize="small">Small Button</Button>
+  </ButtonToolbar>
+  <ButtonToolbar>
+    <Button bsStyle="primary">Regular Button</Button>
+    <Button>Regular Button</Button>
+  </ButtonToolbar>
+  
+  <ButtonToolbar>
+    <Button bsStyle="primary" bsSize="large">Large Button</Button>
+    <Button bsSize="large">Large Button</Button>
+  </ButtonToolbar>
+</Playground>
 
+<!--
 <Playground>
   <ButtonToolbar>
-    <Button bsStyle="primary" bsSize="small">
-      Small button
-    </Button>
+    <Button bsStyle="primary">Standard</Button>
+    <Button>Outline</Button>
+    <Button bsStyle="link">Link</Button>
+    <Button bsStyle="danger">Danger</Button>
+    <Button bsStyle="light">Light</Button>
+    <Button bsStyle="dark">Dark</Button>
   </ButtonToolbar>
 </Playground>
+-->
 
 > Button spacing
 
 > Because React doesn't output newlines between elements, buttons on the same line are displayed flush against each other. To preserve the spacing between multiple inline buttons, wrap your button group in `<ButtonToolbar />`.
 
 ## States
 
-### Block level
+### Mobile Buttons (Block Level)
```

Comment:
The convention of our design system is to have _most_ buttons change to 100% width on mobile devices. I'm certain there will be plenty of exceptions to that. I thought using this block level button would account for that, but maybe it makes more sense for that to be an atomic class?

Reply:
I'm hesitant about the heading saying Mobile as it is misleading as to what the user is likely seeing. We could have some copy below this `Block Buttons` heading to describe how in mobile we would like this style to be used.

For future button designs we should consider defaulting mobile buttons to block style. This would be better than an atomic class, and better than the developer having to manually switch between the two every time a button is used.

Reply:
I'm hesitant about the heading saying Mobile as it is misleading as to what the user is likely seeing. We could have some copy below this `Block Buttons` heading to describe how in mobile we would like this style to be used.

For future button designs we should consider defaulting mobile buttons to block style. This would be better than an atomic class, and better than the developer having to manually switch between the two every time a button is used.

Reply:
It appears there was a merge conflict in need of resolution during commit 5d20750f16b3a24d9dd7f14e7ea8a8a00e0508af.

Reply:
Merge conflict needs to be addressed here.